.chloggen/add_support_for_default_log_collection_receivercreator.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receivercreator
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add support for default annotation-equivalent settings in receivercreator's discovery
```

This sound just a bit confusing. Not sure what `annotation-equivalent` means here. Maybe something like?
```suggestion
note: Add an option to set default annotations to override the discovery logic
```